doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,70 @@
+
+# PaddlePaddle Client
+PaddlePaddle clinet is a command line tool, before startting a cluster train, you need to install it on your latop, you can sumit a cluster train job looks like:
+
+```bash
+paddle k8s job <job-name>
+    --package-path= ./demo
+    --module=demo.train
+    --pserver-count=2
+    --trainer-count=4
+    --output=${OUTPUT_DIR}
+    --input=${INPUT_DIR}
+```
+
+- job-name: you can specify a name for every job, and the name should be uniq.
```

Comment:
Maybe `full_job_name=<k8s-user-name>/<job-name>/<job-id> `? because user will submit the same job name twice.